libs/dal/src/repositories/user/user.schema.ts

```diff
@@ -49,7 +49,7 @@ if (process.env.SELF_HOSTED === 'true') {
     { email: 1 },
     {
       unique: true,
-      partialFilterExpression: { email: 'system@novu.co' },
+      partialFilterExpression: { email: 'hello@example.com' },
```

Comment:
@merrcury why the partialFilterExpression is needed? 🤔

Reply:
Oh, I think it's actually not needed any more. @djabarovgeorge am I correct? (with the new signup approach)

Reply:
removed